Gruntfile.js

```diff
@@ -27,13 +27,25 @@ module.exports = function (grunt) {
             }
         },
         jsonlint: {
-            all: ["package.json", ".jshintrc", "src/**/*.json", "tests/**/*.json", "demos/**/*.json"]
+            all: ["package.json", ".jshintrc", "src/js/*.json", "tests/**/*.json", "demos/**/*.json"]
```

Comment:
In this specific case, Flocking includes a deliberately malformed JSON file as part of its test suite. If this is left as `src/**/.json`, that JSON file is pulled in from `src/lib` and causes a linting error (basically, the linter configuration shouldn't be hitting things in `/lib` because that's an external library directory).